underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystemFactory.java

```diff
@@ -69,20 +69,8 @@ public boolean supportsPath(String path) {
    * @return true if both access and secret key are present, false otherwise
    */
   private boolean addAndCheckOSSCredentials() {
-    String accessKeyConf = PropertyKey.OSS_ACCESS_KEY;
-    if (System.getProperty(accessKeyConf) != null && Configuration.get(accessKeyConf) == null) {
-      Configuration.set(accessKeyConf, System.getProperty(accessKeyConf));
-    }
-    String secretKeyConf = PropertyKey.OSS_SECRET_KEY;
-    if (System.getProperty(secretKeyConf) != null && Configuration.get(secretKeyConf) == null) {
-      Configuration.set(secretKeyConf, System.getProperty(secretKeyConf));
-    }
-    String endPointConf = PropertyKey.OSS_ENDPOINT_KEY;
-    if (System.getProperty(endPointConf) != null && Configuration.get(endPointConf) == null) {
-      Configuration.set(endPointConf, System.getProperty(endPointConf));
-    }
-    return Configuration.get(accessKeyConf) != null
-        && Configuration.get(secretKeyConf) != null
-        && Configuration.get(endPointConf) != null;
+    return Configuration.containsKey(PropertyKey.OSS_ACCESS_KEY)
```

Comment:
the adding logic here is redundant as when initializing `Configuration` we already load System properties to `Configuration`.